install.sh

```diff
@@ -60,6 +60,7 @@ execute() {
     fi
     install "${srcdir}/${binexe}" "${BINDIR}/"
     log_info "installed ${BINDIR}/${binexe}"
+    log_info "Add ${BINDIR} to PATH variable"
```

Comment:
@lekaf974 Sorry for getting back a little late on this one. I think instead of we asking to update the PATH on the user system, we can change the message and let them decide where they want to move. 

So, I think you change of installing karmor in `./bin/karmor` looks good to me and I will change the message to something like `karmor is installed in ./bin/karmor` and `invoke ./bin/karmor or move karmor to your desired PATH` 

It's an open discussion and we can discuss and agree on what works best. 
Let me know if you've any follow-up questions here.

Reply:
LGTM I don't have a real opinion, I'll update the message so the user can decide what he want to do soon!

Reply:
Latest output
```bash
kubearmor/kubearmor-client info checking GitHub for latest tag
kubearmor/kubearmor-client info found version: 1.2.1 for v1.2.1/linux/amd64
kubearmor/kubearmor-client info installed ./bin/karmor
kubearmor/kubearmor-client info karmor is installed in ./bin
kubearmor/kubearmor-client info invoke ./bin/karmor or move karmor to your desired PATH
```

Reply:
@kranurag7 ready for another review

Reply:
From my side, this looks good to me. Thank you for working on this.